pkg/terraform/exec/Gopkg.toml

```diff
@@ -27,6 +27,10 @@ ignored = [
   revision = "232e45548389bd9357411a6922a07c5fd4068bda"
   name = "golang.org/x/oauth2"
 
+[[override]]
```

Comment:
you should be able to add `golang.org/x/net` to override here https://github.com/openshift/installer/pull/1555/commits/2237e3ce045212f958f7837864461ea0f2b40dd0#diff-40d918378d3e037d5cabc171f3e2065eR2 so that dep ignored dependency on that package.

Reply:
I don't understand your comment. it is there. that's what I did no?

Reply:
with your current change:
```console
(pr-1555) $ cd pkg/terraform/exec

## try to update the golang.org/x/oauth2 which internally need golang.org/x/net
(pr-1555) $ dep ensure -update golang.org/x/oauth2
(pr-1555) $ gst
On branch pr-1555
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   Gopkg.lock

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        vendor/golang.org/x/net/

## this brought the golang.org/x/net back into vendor and also updated the lock file.
(pr-1555) $ git --no-pager diff
diff --git a/pkg/terraform/exec/Gopkg.lock b/pkg/terraform/exec/Gopkg.lock
index 3bca13287..303fb1a36 100644
--- a/pkg/terraform/exec/Gopkg.lock
+++ b/pkg/terraform/exec/Gopkg.lock
@@ -1133,6 +1133,25 @@
   pruneopts = "NUT"
   revision = "505ab145d0a99da450461ae2c1a9f6cd10d1f447"

+[[projects]]
+  digest = "1:42ed38949b3ccadeb4718632a74ae17ae5b4ea86e79e6ee85db8fea64e73147d"
+  name = "golang.org/x/net"
+  packages = [
+    "context",
+    "context/ctxhttp",
+    "html",
+    "html/atom",
+    "html/charset",
+    "http/httpguts",
+    "http2",
+    "http2/hpack",
+    "idna",
+    "internal/timeseries",
+    "trace",
+  ]
+  pruneopts = "NUT"
+  revision = "eb5bcb51f2a31c7d5141d810b70815c05d9c9146"
+
 [[projects]]
   digest = "1:4648e14100dfb6fde9b6caa98d3b486721dfdf895cca58e3950240d5ef051f7b"
   name = "golang.org/x/oauth2"
```

what I was recommending was 
```console
(pr-1555) $ git --no-pager diff Gopkg.toml
diff --git a/pkg/terraform/exec/Gopkg.toml b/pkg/terraform/exec/Gopkg.toml
index 56ef2ab89..2b140df4f 100644
--- a/pkg/terraform/exec/Gopkg.toml
+++ b/pkg/terraform/exec/Gopkg.toml
@@ -1,5 +1,6 @@
 ignored = [
-  "github.com/openshift/installer/pkg/terraform/exec/plugins*"
+  "github.com/openshift/installer/pkg/terraform/exec/plugins*",
+  "golang.org/x/net*", ## there can only be one copy of golang.org/x/net in the binary and that copy is maintained in the root vendor.
 ]

 [prune]
@@ -27,10 +28,6 @@ ignored = [
   revision = "232e45548389bd9357411a6922a07c5fd4068bda"
   name = "golang.org/x/oauth2"

-[[override]]
-  revision = "eb5bcb51f2a31c7d5141d810b70815c05d9c9146"
-  name = "golang.org/x/net"
-
 [[override]]
   revision = "cc9d70694ef317c2e6443e20b7927363d69c8b1e"
   name = "github.com/hashicorp/go-slug"
```

Reply:
@abhinavdahiya Did I miss something?

Reply:
> you should be able to add golang.org/x/net to override here

So what you meant to write was to add it to the` ignored` as well as to the `override`?

Reply:
fixed